package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -81,52 +81,11 @@
 HydrogenBondLifetimes
 ~~~~~~~~~~~~~~~~~~~~~
 
-Analyzing hydrogen bond lifetimes (HBL) :class:`HydrogenBondLifetimes`, both
-continuos and intermittent. In this case we are analyzing how residue 38
-interact with a water sphere of radius 6.0 centered on the geometric center of
-protein and residue 42. If the hydrogen bond lifetimes are very stable, we can
-assume that residue 38 is hydrophilic, on the other hand, if the are very
-unstable, we can assume that residue 38 is hydrophobic::
+The class Analyzing hydrogen bond lifetimes (HBL)
```

Comment:
Instead of saying that it has been removed just say that there are other ways to analyze water dynamics is the ...lifetimes() method. 

Add a See Also section for the hbond_analysis module.